### PR TITLE
refactor: move components, rename folders

### PR DESCRIPTION
**Pull request by Sebastian-ubs**
This change is [Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1026)

grafik

grafik

### CODE REVIEW

**Review by jolierabideau — COMMENTED** [review]
Thanks Sebastian! Just some path aliases need to be added back

File: lib/platform-bible-react/src/components/basics/search-bar.component.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useState } from 'react';
-import TextField from '@/components/text-field.component';
-import '@/components/search-bar.component.css';
+import TextField from './text-field.component';
+import './search-bar.component.css';
```
[the remark this answers is not in the document]
Comment (Sebastian-ubs) [reply lib/platform-bible-react/src/components/basics/search-bar.component.tsx]: done

File: lib/platform-bible-react/src/components/basics/tabs-vertical.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import React from 'react';
 import * as TabsPrimitive from '@radix-ui/react-tabs';
 import { cn } from '@/utils/shadcn-ui.util';
-import { TabsContentProps, TabsListProps, TabsTriggerProps } from './tabs';
+import { TabsContentProps, TabsListProps, TabsTriggerProps } from '../shadcn-ui/tabs';
```
[the remark this answers is not in the document]
Comment (Sebastian-ubs) [reply lib/platform-bible-react/src/components/basics/tabs-vertical.tsx]: done

File: lib/platform-bible-react/src/components/basics/text-field.component.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import { Input as ShadInput } from '@/components/shadcn-ui/input';
-import { Label as ShadLabel } from '@/components/shadcn-ui/label';
+import { Input as ShadInput } from '../shadcn-ui/input';
```
[the remark this answers is not in the document]
Comment (Sebastian-ubs) [reply lib/platform-bible-react/src/components/basics/text-field.component.tsx]: done

File: lib/platform-bible-react/src/components/mui/checkbox.component.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { FormLabel, Checkbox as MuiCheckbox } from '@mui/material';
 import { ChangeEvent } from 'react';
-import '@/components/checkbox.component.css';
-import LabelPosition from '@/components/label-position.model';
+import './checkbox.component.css';
+import LabelPosition from './label-position.model';
```
[the remark this answers is not in the document]
Comment (Sebastian-ubs) [reply lib/platform-bible-react/src/components/mui/checkbox.component.tsx]: done

File: lib/platform-bible-react/src/components/mui/gridMenu.component.onClick.test.tsx [hunk 5]
```diff
@@ -1,9 +1,9 @@
 import { NonValidatingDocumentCombiner, PlatformMenus } from 'platform-bible-utils';
 import { fireEvent, render, screen } from '@testing-library/react';
 import '@testing-library/jest-dom';
-import GridMenu from '@/components/grid-menu.component';
-import { Command } from '@/components/menu-item.component';
-import * as jsonMenu from '@/components/sample.composed.full.menu.json';
+import GridMenu from './grid-menu.component';
+import { Command } from './menu-item.component';
+import * as jsonMenu from './sample.composed.full.menu.json';
```
[the remark this answers is not in the document]
Comment (Sebastian-ubs) [reply lib/platform-bible-react/src/components/mui/gridMenu.component.onClick.test.tsx]: resolved

**Review by Sebastian-ubs — COMMENTED** [review]
done for all + optimized imports

**Review by jolierabideau — DISMISSED** [review]

**Comment by Sebastian-ubs** [comment]
rebased on main and resolved conflicts

**Review by jolierabideau — APPROVED** [review]
Thanks Sebastian!

**Comment by Sebastian-ubs** [comment]
Thanks @jolierabideau, looks like reviewable is holding us back

**Review by jolierabideau — APPROVED** [review]
> Reviewed 1 of 69 files at r1, 3 of 27 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1026)* status: 4 of 76 files reviewed, 5 unresolved discussions

**Review by jolierabideau — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1026)* status: 4 of 76 files reviewed, all discussions resolved

**Review by jolierabideau — APPROVED** [review]
> Reviewed 41 of 69 files at r1, 24 of 27 files at r2, 7 of 7 files at r3.
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1026)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved

**Pull request merged**